### PR TITLE
Move type definitions into build dirs

### LINKED ISSUE #646: Move type definitions into `./build` and out of VCS
After #616

### PR DESCRIPTION
**Pull request by willclarktech**
Closes #646

### CODE REVIEW

File: packages/cosmwasm/package.json [hunk 1]
```diff
@@ -29,11 +28,10 @@
     "format-text": "prettier --write --prose-wrap always --print-width 80 \"./*.md\"",
     "lint": "eslint --max-warnings 0 \"**/*.{js,ts}\"",
     "lint-fix": "eslint --max-warnings 0 \"**/*.{js,ts}\" --fix",
-    "move-types": "shx rm -rf ./types/* && shx mv build/types/* ./types",
-    "format-types": "prettier --write --loglevel warn \"./types/**/*.d.ts\"",
+    "remove-test-types": "shx rm -rf build/*.spec.d.ts build/*/*.spec.d.ts",
```
Comment (webmaster128) [inline packages/cosmwasm/package.json:31]: is this removal still needed/useful?

**Review by webmaster128 — COMMENTED** [review]
Could you have a look into `packages/stargate/.gitignore`? There we have some pattern instead of just `build/`. Do we need to streamline other .gitignore files as well?

**Comment by willclarktech** [comment]
> There we have some pattern instead of just build/. Do we need to streamline other .gitignore files as well?

I believe this is just a hangover from when I was explicitly including files that were in the `build` directory.

**Review by webmaster128 — APPROVED** [review]
